### ISSUE TITLE
Mac Dedicated GPU activated when launch warp

### ISSUE THREAD

**Issue by rodrigotrombeta** [warpdotdev/Warp#2114]
### Describe the bug

Dear team,

In my tests using a macbook 16 inch model 2019 I have noticed that once I open warp the dedicated GPU (AMD Raedon) gets enabled and the Fans starts to spin higher because of that (needless to say that the notebook  temperature also considerable increase).

Using warp for a while shows that the process "stable" is the responsible for turning on the dedicated GPU ( shown by gfxCardStatus ( https://gfx.io )).

That's a problem in that computer model (when the GPU is enabled the Fan also turns on and the heat increases), but would it be possible to enable warp without using the dedicated graphics card?

### To Reproduce

Open warp in a macbook pro 16 inch model 2019. i7 CPU.

### Expected behaviour

Expected is that a shell terminal does not need to use an external GPU.

### Operating System

MacOS

### OS Version

Ventura 13.0

### Shell Version

zsh 5.8.1 (x86_64-apple-darwin22.0)

### Warp Version

v0.2022.11.08.08.07.stable_02

### Does this block you from using Warp daily?

Yes

**Comment by rodrigotrombeta** [comment]
Looks like that's a feature with a clear purpose:

https://www.warp.dev/faq#why-did-you-build-warp-to-be-gpu-accelerated

Unfortunately macbook pro 16 inch is very well known to suffer a lot when turning on the external GPU.

**Comment by kevinyang372** [comment]
Hey @rodrigotrombeta Sorry just saw this issue. A similar issue has been reported a while back: https://github.com/warpdotdev/Warp/issues/76

We are still monitoring this to decide if this is something we want to prioritize on. Feel free to upvote / comment there if this is import to you!

**Comment by rodrigotrombeta** [comment]
Hello,

After research that (known issue) I've found the following discussion:
https://discussions.apple.com/thread/8160651?answerId=32574288022#32574288022

So in order to check if the external GPU is being used:
`pmset -g | grep -i gpu`

Result must be 0

If need to turn off the external GPU (and force Macbook to use the integrated one) then issue:
`sudo pmset -a gpuswitch 0`

The options are:
(https://usunyu.com/note/2020/08/21/switch-integrated-graphics-when-using-battery-macos/)

0 - integrated GPU only
1 - discrete GPU only
2 - autoswitch GPU

Hope I've helped.
This is a normal complain on Macbook 16, so unfortunately that's an alternative to be able to work in peace of mind.

I'm testing in order to see if it works fine and cause no issues.

Kind regards,
Rodrigo

**Comment by gkrawiec** [comment]
For the past 3 weeks I've had a battery and  noisy fan issue on my MBP16 2019. I spent hours debugging it. Unfortunately, I had just upgraded macOS and also firefox....and extensions update automatically, so I couldn't concentrate on anything specifically.. Hours and hours wasted trying to figure this out. disabling extensions, reinstalling firefox, etc. Nothing worked. Today I realized it's warp terminal that forces the discreet gpu to stay on.  This requirement is beyond ridiculous. Uninstalled it for the time being. Hopefully you guys can figure out how to display a line of text without taking 30% off my battery.
